types/twilio/index.d.ts

```diff
@@ -1,6 +1,8 @@
-// Type definitions for twilio
+// Type definitions for twilio 2.11
```

Comment:
Current version is 3.39 on npm. Any clue on significant changes? Should it just be marked as 3.39?

Reply:
Twilio 3+ has its own typings defined in the repo/npm package.
https://github.com/twilio/twilio-node/blob/master/package.json#L63

And yes this is majorly different than Twilio v3. They pretty much did a complete rewrite to the library.

So my intentions is to fix up the typings for the version of Twilio I'm using since either upgrading TypeScript to the latest version or upgrading my node version to node 12 (and thusly the @types/node) caused an error with these Typings. Not sure since I upgraded TypeScript and Node at the same time.

Reply:
Gotcha, thanks for the response!